src/bms_main.cpp

```diff
@@ -65,11 +65,17 @@ void bms_setup() {
 
   relay->setPowerOffCallback([]() {
     Settings->graceful_shutdown_count++;
+    Settings->mah_state = relay->getChargeStateMah();
+    //Settings->mah_max = relay->batteryCapacityEstimate();
     saveSettings();
   });
 
   relay->setBMSSerialOverride(0xFFABCDEF);
 
+  relay->setChargeStateMah(Settings->mah_state);
```

Comment:
potentially we may want to remove this setting altogether and replace it with an estimate that comes from the voltage on every startup.  This will help prevent tracking drift that may occur if users dont charge to full often enough.  However I'm not sure we have voltages available this early, so this should work for now.

Reply:
I think this should be fine. If someone hasn't charged their wheel for a bit they can just put it to charge to reset the state no? If you get it from voltage every time there might be some discrepancy built in for those who don't leave it off for too long.

Reply:
I agree, probably not a huge issue, but its nice to have.  I've found that charging on a speed charger causes about a 10-15% drift between what OWIE thinks was regained vs what was actually regained, so this drift can happen fast if you don't fill up completely often.

Reply:
That drift has made me wonder if the 0.055 scaling factor used for current is actually correct. Maybe it being slightly off adds up to a noticeable error over time. How did that value get determined?